src/libraries/Microsoft.Extensions.Diagnostics.Abstractions/src/Metrics/InstrumentRule.cs

```diff
@@ -20,32 +20,41 @@ namespace Microsoft.Extensions.Diagnostics.Metrics
     /// <param name="meterName">The <see cref="Meter.Name">Meter.Name</see> or prefix.</param>
     /// <param name="instrumentName">The <see cref="Instrument.Name">Instrument.Name</see>.</param>
     /// <param name="listenerName">The <see cref="IMetricsListener.Name">IMetricsListener.Name</see>.</param>
-    /// <param name="scopes">The <see cref="MeterScope"/>'s to consider.</param>
-    /// <param name="enable">Enables or disabled the matched instrument for this listener.</param>
+    /// <param name="scopes">A bitwise combination of the enumeration values that specifies the scopes to consider.</param>
+    /// <param name="enable"><see langword="true" />to enable the matched instrument for this listener; otherwise, <see langword="false" />.</param>
```

Comment:
```suggestion
    /// <param name="enable"><see langword="true" /> to enable the matched instrument for this listener; otherwise, <see langword="false" />.</param>
```